docs/source/backend/do_import.rst

```diff
@@ -0,0 +1,7 @@
+do\_import module
+=================
+
+.. automodule:: do_import
```

Comment:
This page is [empty](https://os2mo.readthedocs.io/projects/phonebook/en/feature-25304_documentation_and_sphinx/backend/do_import.html)?

Reply:
No longer!